extensions/8to16/CTDB.js

```
@@ -0,0 +1,96 @@
+
+// this is basically the same extension as https://ctdbapi.funstrangeegg.workers.dev/extension.js
+// (which is made for CodeTorch)
+
+(function (Scratch) {
+  "use strict";
+
+  class WWDBExtension {
+    getInfo() {
+      return {
+        id: "ampctdbapi",
+        name: "Worldwide DB",
```

ESLint

[error] Extension name should usually be translated

(extension/should-translate)